src/createServer.js

```diff
@@ -1,3 +1,60 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { URL } = require('url')
+const {convertToCase } = require('./convertToCase/convertToCase');
+
+const availableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+function createServer() {
+  return http.createServer((req, res) => {
+    res.setHeader('Content-type', 'application/json');
+
+    const newUrl = new URL(req.url, `http://${req.headers.host}`);
+
+    const targetCase = newUrl.searchParams.get('toCase');
+    const textToConvert = newUrl.pathname.slice(1);
+
+    const isError = !textToConvert
```

Comment:
The condition for checking isError could be simplified. Instead of using every and !==, you can use includes to check if targetCase is in the availableCases array.